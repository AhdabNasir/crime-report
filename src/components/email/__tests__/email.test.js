import React from 'react';
import renderer from 'react-test-renderer';
import Email from '..';

describe('<Email/>', () => {
  it('renders <Email/> component', () => {
    const tree = renderer
      .create(<Email emailAddress="test@email.com" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
