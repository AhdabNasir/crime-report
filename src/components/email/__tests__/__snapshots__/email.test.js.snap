// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Email/> renders <Email/> component 1`] = `
<a
  className="govuk-link"
  href="mailto:test@email.com"
>
  test@email.com
</a>
`;
